en/docs/next/ai-workspace/getting-started.md

```
@@ -85,6 +103,63 @@ An LLM provider connects AI Workspace to an AI service platform such as OpenAI,
+If you don't run `setup.sh`, provision the at-rest encryption key yourself before the first start. It protects [AI Workspace secrets](secrets-management.md), subscription tokens, and WebSub HMAC secrets, and the Platform API refuses to start if it's missing or malformed. Keep it stable across restarts and replicas.
+
+The key is a single 32-byte AES-256 value, supplied as 64 hex characters or base64. Generate it and write it to the file the container mounts at `/etc/platform-api/keys`:
+
+```sh
+openssl rand -hex 32 > resources/keys/encryption.key
+```
+
+A trailing newline is trimmed on load. The Platform API doesn't read the key from an environment variable directly. It reads the `encryption_key` field in `config.toml`, which pulls the value in through an interpolation token:
+
+{% raw %}
+```toml
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)